test/integration/collection-view.spec.js

```diff
@@ -0,0 +1,139 @@
+const _ = Cypress._;
+const $ = Cypress.$;
+
+describe('CollectionView', () => {
+  let BbModel;
+  let BbCollection;
+  let CollectionView;
+  let MnView;
+
+  beforeEach(() => {
+    cy.server();
+
+    cy.visitMn(function({Marionette, Backbone}) {
+      BbModel = Backbone.Model;
+      BbCollection = Backbone.Collection;
+      MnView = Marionette.View;
+
+      const FooView = Marionette.View.extend({
+        template: _.template('Foo template content')
+      });
+
+      const BarView = Marionette.View.extend({
+        template: _.template('Bar template content')
+      });
+
+      CollectionView = Marionette.CollectionView.extend({
+        className: 'collection-view-container',
```

Comment:
this might be better as `id: 'collection-view-container',` since there should be only one instance.